#Base
logger/Event base
#props
uint16 mision_id
